packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -1069,20 +1071,26 @@ function CheckoutLineItem( {
 	const productSlug = product?.product_slug;
 
 	const label = getLabel( product );
+	const itemOriginalSubtotalInteger =
+		isAkPro500Cart && product.quantity
+			? product.item_original_cost_for_quantity_one_integer
+			: product.item_original_subtotal_integer;
+	const originalAmountDisplay = formatCurrency( itemOriginalSubtotalInteger, product.currency, {
+		isSmallestUnit: true,
+		stripZeros: true,
+	} );
+	const originalAmountInteger = itemOriginalSubtotalInteger;
```

Comment:
I don't think we need this variable anymore if it is always equal to `itemOriginalSubtotalInteger`, and seeing as they are both defined as constants, they should always be equal

Reply:
Yes, thats for catching that. 👍